src/styles/index.css

```diff
@@ -0,0 +1,18 @@
+/* Error: Undefined variable.
+ *   ,
+ * 4 |   gap: $gap;
```

Comment:
The error message indicates that there is an undefined variable `$gap`. Ensure that the variable `$gap` is defined in your SCSS files before it is used.